addOns/pscanrulesAlpha/src/main/zapHomeFiles/xml/js-function-list.txt

```diff
@@ -0,0 +1,10 @@
+# This file lists all dangerous JS functions being scanned for.
+# Searching is done via regex per whole word match on each line item [e.g. 'eval' will match with '$eval()'].
```

Comment:
Is the regex part relevant? That seems to be an implementation detail, or you mean these are to be interpreted as regular expressions as well? If the former we should quote the strings when creating the `Pattern` otherwise we should catch the exception when the regular expression is invalid.

Reply:
Hmm I think we can omit the word "regex", I wasn't intending for user defined regex expressions in the payload. I just wanted to be more verbose with explaining how the searching would happen.

Reply:
OK.